pageserver/src/tenant.rs

```diff
@@ -2912,7 +2915,7 @@ impl Tenant {
         };
         // create a `tenant/{tenant_id}/timelines/basebackup-{timeline_id}.{TEMP_FILE_SUFFIX}/`
         // temporary directory for basebackup files for the given timeline.
-        let initdb_path = path_with_suffix_extension(
+        let pgdata_path = path_with_suffix_extension(
```

Comment:
Would prefer this rename to happen in a preliminary PR so the essence of this PR doesn't get lost in the curn of the renames.